wordpress-mobile/WordPress-iOS
# Issue/12592 Update fancy button border style

Fixes #12592. This PR pulls in updates to WordPressUI and WordPressAuthenticator to use the new styles for fancy buttons.

![fancy-button-before-after](https://user-images.githubusercontent.com/4780/66329317-3cd2fc80-e926-11e9-9247-d8e595d2e39b.png)

**To test:**

* Build and run. Test on iOS 13 light and dark mode, and iOS 12.
* Step through the NUX flow and check the appearance of both primary and secondary buttons.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary (bundling this under dark mode improvements).
